### PR TITLE
feat: create loader component gf-9

### PR DESCRIPTION
**Pull request by perekotypole**
Implemented a loader component:
- pulsing animation (similar to https://www.davidhu.io/react-spinners/storybook/?path=/docs/puffloader--docs)

image

### CODE REVIEW

File: apps/frontend/src/libs/components/loader/loader.tsx
Comment (liza-veis) [inline apps/frontend/src/libs/components/loader/loader.tsx:1]: Can we add something more engaging, like this?

Screenshot 2024-08-19 at 11 50 07

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/components/loader/loader.tsx
Comment (perekotypole) [reply apps/frontend/src/libs/components/loader/loader.tsx:1]: I have changed it to another kind of animation (link and screenshot in the PR description). If it's not okay, I will replace it to spinner

Comment (liza-veis) [reply apps/frontend/src/libs/components/loader/loader.tsx:1]: Cool, I like the new one

File: apps/frontend/src/libs/components/app/app.tsx [hunk 1]
```diff
@@ -47,11 +47,16 @@ const App = (): JSX.Element => {
 				<>
 					<h2>Users:</h2>
 					<h3>Status: {dataStatus}</h3>
-					<ul>
-						{users.map((user) => (
-							<li key={user.id}>{user.email}</li>
-						))}
-					</ul>
+
+					{dataStatus === "pending" ? (
```
Comment (liza-veis) [inline apps/frontend/src/libs/components/app/app.tsx:51]: move this to a constant
```suggestion
					{isLoading ? (
```

File: apps/frontend/src/libs/components/loader/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+:root {
+	--size: 50px;
+	--weight: 5px;
+	--animation-time: 2s;
+}
```
Comment (liza-veis) [inline apps/frontend/src/libs/components/loader/styles.module.css:5]: Let's remove variables that are used only once; for animation-duration, it's better to keep this in .loader scope, not in the root
```suggestion
.loader {
	--animation-duration: 2s;
}
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by liza-veis — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**